cmake/treedata/common/subdirs.txt

```diff
@@ -1,16 +1,23 @@
 xbmc                                            xbmc
 xbmc/addons                                     addons
 xbmc/addons/binary-addons                       addons_binary-addons
-xbmc/addons/interfaces                          addonsBinaryInterfaces
-xbmc/addons/interfaces/Addon                    addonCallbacks_Addon
+xbmc/addons/interfaces                          addons_interfaces
+xbmc/addons/interfaces/Addon                    addons_interfaces_addon
```

Comment:
any reason for those changes?

Reply:
Cosmetics, to match the real directory names. They were renamed some time ago...